<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scholar</class>
 <widget class="QMainWindow" name="Scholar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Scholar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="ScholarGuitarSignal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="ScholarControlFrame">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="ScholarPlayState">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Paused</string>
     </property>
    </widget>
    <widget class="QSlider" name="ScholarBPM">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>20</number>
     </property>
     <property name="maximum">
      <number>240</number>
     </property>
     <property name="value">
      <number>80</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="ScholarBPMlbl">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>BPM:</string>
     </property>
    </widget>
    <widget class="QLabel" name="ScholarMetronomeLbl">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>0</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Metronome</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ScholarMetronomeOn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>On</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ScholarMetronomeOff">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ScholarBPMvalue">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string>990; </string>
     </property>
     <property name="text">
      <string extracomment="20..240"/>
     </property>
     <property name="placeholderText">
      <string>20..240</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="ScholarMainView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>991</width>
      <height>281</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="interactive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="ScholarSoundEffecrFrame1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>350</y>
      <width>71</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="ScholarSoundEffecrFrame2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>350</y>
      <width>71</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="ScholarSoundEffecrFrame3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>350</y>
      <width>71</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QSlider" name="ScholarVolumeLvl">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>350</y>
      <width>20</width>
      <height>91</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QFrame" name="ScholarBgSoundFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="ScholarBgSoundOn">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>On</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ScholarBgSoundOff">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="ScholarBGMusicLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>BG music:</string>
     </property>
    </widget>
    <widget class="QLabel" name="ScholarBGMusicNameLbl">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="ScholarAudioVisualHelperFrame">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>40</y>
      <width>231</width>
      <height>271</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionTuner"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTuner">
   <property name="text">
    <string>Tuner</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>Scholar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
